src/test/java/org/openmrs/performance/scenarios/VisitPatientScenario.java

```diff
@@ -35,6 +35,7 @@ public ScenarioBuilder getScenarioBuilder() {
 				.exec(registry.openImmunizationsTab("#{patient_uuid}"))
 				.exec(registry.openAttachmentsTab("#{patient_uuid}"))
 				.exec(registry.addDrugOrder("#{patient_uuid}", "#{visitUuid}", "#{currentUserUuid}"))
+				.exec(registry.addVisitNote("#{patient_uuid}", "#{currentUserUuid}"))
 				.exec(registry.endVisit("#{patient_uuid}"));
```

Comment:
Let's add some pauses to make it realistic. In real world, users takes time to perform actions. Also this may fix the `Cannot contain encounters whose dates are after the stop of the visit.` error. 

```suggestion
				.exec(registry.addDrugOrder("#{patient_uuid}", "#{visitUuid}", "#{currentUserUuid}"))
				.pause(5)
				.exec(registry.addVisitNote("#{patient_uuid}", "#{currentUserUuid}"))
				.pause(10)
				.exec(registry.endVisit("#{patient_uuid}"));
```